src/Services/FieldResolver.php

```diff
@@ -103,6 +103,11 @@ public function resolve($source, $args, $context, ResolveInfo $info)
             }
         }
 
+        if (!empty($fieldConfig['sanitize'])) {
+            $value = $property ?? '';
+            $property = htmlspecialchars($value);
```

Comment:
```suggestion
            return htmlspecialchars($value);
```

@prajapati-kaushik I'm changing this code to apply early returns, like the other code in this function: https://freek.dev/1593-writing-readable-php-decrease-indentation-by-returning-early